manifests/rule.pp

```diff
@@ -49,7 +49,7 @@
 # @param daddr The destination address we want to match
 # @param proto_options Optional parameters that will be passed to the protocol (for example to match specific ICMP types)
 # @param interface an Optional interface where this rule should be applied
-# @param ensure Set the rule to present or absent
+# @param outerface an Optional Outerface to match egress packets
```

Comment:
please readd the line you deleted :)

Reply:
Whoops, Sorry, Suspect I might have highlighted it with the laptop track pad and not noticed